paddle/fluid/platform/enforce.h

```diff
@@ -230,13 +230,13 @@ inline std::string SimplifyDemangleStr(std::string str) {
 }
 
 inline std::string GetCurrentTraceBackString() {
-  static constexpr int TRACE_STACK_LIMIT = 100;
+  //static constexpr int TRACE_STACK_LIMIT = 100;
```

Comment:
I think here we can use `#ifdef PADDLE_WITH_MUSL` control the backtrace related code, now paddle still need backtrace in many cases.

Reply:
ok

Reply:
ready in 2e12aebf246b69294ff57ee797105b43ab288b0e